pynestml/meta_model/ast_input_line.py

```diff
@@ -48,14 +48,14 @@ class ASTInputLine(ASTNode):
         signal_type = None
     """
 
-    def __init__(self, name=None, size_parameter=None, data_type=None, input_types=list(), signal_type=None,
+    def __init__(self, name=None, size_parameter=None, data_type=None, input_types=None, signal_type=None,
```

Comment:
is it just me or are we missing an initialization of ``input_types`` to ``[]`` at the beginning of the function now?

Reply:
This is a common Python pitfall/antipattern; see e.g. https://nikos7am.com/posts/mutable-default-arguments/

Reply:
Misread your comment; you are right! Thanks for catching this.